From 5196ae669a42bbb92b5c4042f2a5087db90879c5 Mon Sep 17 00:00:00 2001
From: Jon Turney <jon.turney@dronecode.org.uk>
Date: Wed, 9 Apr 2014 21:14:07 +0100
Subject: [PATCH 16/29] Use __USE_MINGW_ANSI_STDIO for MinGW build

Use __USE_MINGW_ANSI_STDIO for ANSI C99 compatible implementation of printf()
and friends on MinGW

Perhaps we don't need this, if all the printf formats used are in the subset
supported by MSVCRT, but it's easier to turn this on than audit every
single format string...

Signed-off-by: Jon Turney <jon.turney@dronecode.org.uk>
---
 Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.am b/Makefile.am
index b71f4b73..39aec328 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -59,6 +59,7 @@ endif
 if MINGW_HOST
 SOCKET_LIBS = -lws2_32
 AM_CPPFLAGS += -D_POSIX_C_SOURCE
+AM_CPPFLAGS += -D__USE_MINGW_ANSI_STDIO
 endif
 
 # Specify include paths for ac macros
-- 
2.15.0

